Libraries/acdriver/CMakeLists.txt

```diff
@@ -14,11 +14,17 @@ add_library(acdriver SHARED
             Sources/Result.cpp
             Sources/VersionAction.cpp
             Sources/CompileAction.cpp
+            Sources/CompileActionImageSet.cpp
+            Sources/CompileOutput.cpp
```

Comment:
I think it'd probably be good to add this in a separate commit (maybe even PR) with tests. Should be straightforward to test, I think.